config/config.test.json

```diff
@@ -13,5 +13,13 @@
         "BASE_URL": "http://127.0.0.1:8080",
         "REALM_NAME": "master",
         "CLIENT_ID": "admin-cli"
+    },
+    "SCHULPORTAL": {
```

Comment:
Just use the Keycloak-Config that is already there

Reply:
now the existing Keycloak-Config is used for realm and client-id of Schulportal